ateam_ssl_simulation_radio_bridge/src/ssl_simulation_radio_bridge_node.cpp

```diff
@@ -58,7 +58,7 @@ class SSLSimulationRadioBridgeNode : public rclcpp::Node
 
       command_subscriptions_.at(robot_id) =
         create_subscription<ateam_msgs::msg::RobotMotionCommand>(
-        "~/robot_motion_commands/robot" + std::to_string(robot_id),
+        "/ateam_ai/robot_motion_commands/robot" + std::to_string(robot_id),
```

Comment:
I'm curious what motivated this change instead of remapping topics in the associated launch file?

Reply:
Honestly, I figured that with 16 robots, I mess up less just editing the sub topic instead of trying to copy paste that remapping 16 times. All the other pub sub relationships are hardcoded as well. I don't know the "right" way to do this in ROS, so I very open to change it

Reply:
I _believe_ you can remap base names for multiple topics in one remap rule in ROS 2, though I can't remember if I've tried that before.

There are two popular conventions for handling topic names, though there's a ton of code that doesn't follow either out there in the world, since nothing enforces it.
- Nodes default to global topic names, such as `/cmd_vel` and `/image_raw`, which are then remapped to specific names, such as `/camera/front/color/image_raw`
   Under this pattern, we'd use `/robot_motion_commands/robot0`
- Nodes default to private topic names, such as `/my_controller/cmd_vel`, which are then remapped to specific names.
   Under this pattern, we'd use `~/robot_motion_commands/robot0`

Of course, this is all a little arbitrary since everything works as long as you sort out the names in the end. I commented mostly out of curiosity if you had run into some problem with using the locally namespaced name.

Reply:
I gotcha. In the long term I think it's going to be required that we refactor this into a cleaner convention, but I don't think it's required right this second. I'll make an issue for it so we can track the work